earth_enterprise/src/fusion/portableglobe/servers/local_server.py

```diff
@@ -506,8 +517,13 @@ def LocalJsonHandler(self, handler, is_2d=False):
       json_end += "};"
 
     # Adding globe name helps ensure clearing of cache for new globes.
-    handler.write("%s/%s%s" % (
-        json_start, tornado.web.globe_.GlobeShortName(), json_end))
+    json_text = "%s/%s%s" % (
+      json_start, tornado.web.globe_.GlobeShortName(), json_end)
+
+    if json_version > 1:
```

Comment:
Minor: might be nice to check explicitly for '2' here in case we want to add other versions in the future.

Reply:
I thought about this a bit!  That's what it was originally.  My reasoning for going for version > 1 instead is that, in the event that later versions are added, one of two cases are probably true:

1.  They want to do something totally different at this particular point, in which case they'll be editing it here and replacing this check.

2.  They want to do something different elsewhere based on the version number, in which case the "expected" result of incrementing the version is probably that they want to build on top of version 2 rather than regressing to version 1.

Reply:
That's sound reasoning. Works for me.

Reply:
if one pass v=3 (or any number > 1) they will get the new functionality while it was only intended for specific version (2). I think having specific version would resolve this issue and if future version support is needed, code would need to be changed any way.